test/cljs/status_im/test/utils/varint.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.test.utils.varint
+  (:require [cljs.test :refer-macros [deftest is testing]]
+            [status-im.utils.varint :as varint]))
+
+(deftest encode
+  (is (= (varint/encode-hex 0x0)
+         "0"))
+  (is (= (varint/encode-hex 0x70)
+         "70"))
+  (is (= (varint/encode-hex 0xe3)
+         "e301")))
+
+(deftest decode
+  (is (= (varint/decode-hex "0")
+         0x0))
+  (is (= (varint/decode-hex "70")
+         0x70))
+  (is (= (varint/decode-hex "e301")
+         0xe301)))
+
+(deftest roundtrip
+  (is (= (varint/decode-hex (varint/encode-hex 0))
+         2684453)
```

Comment:
could you elaborate on this ?

Reply:
it's just a roundtrip test but it is wrong thanks I fixed it